charts/spire/charts/spire-server/templates/tornjak-config.yaml

```diff
@@ -39,13 +39,62 @@ data:
         }
       }
     {{- end }}
+
+      SPIRECRDManager {
+        plugin_data {
+          classname = "{{ .Values.controllerManager.className }}"
+        }
+      }
+
```

Comment:
hmm....

Is the crd manager functionality disable-able? If so, is it leaving this section out?

Does it use the permissions of the k8s service account in order to manage the crds?

Reply:
Back when it was under development, Tornjak shared the same pod as the controller manager, and spire server (IIRC). Then, it inherited all of the permissions of the controller manager which is what is necessary. Has this changed?